### PR TITLE
Snap: overview, preparation, interactive demo

### PR DESCRIPTION
**Pull request by rashmikampurath**
@Anuja: Snap files for internal review.

### CODE REVIEW

**Review by AnujaChangede — COMMENTED** [review]
Please make the suggested changes.

**Comment by rashmikampurath** [comment]
@rizdaprasetya /snap/overview.md, /snap/preparation.md, /snap/interactive-demo.md are ready for your technical review. Kindly go through it and suggest changes and corrections.

File: en/snap/interactive-demo.md [hunk 1]
```diff
@@ -1,46 +1,47 @@
...
 </div>
 
-### Technology Stack
-
-The sample integration's:
-- **Backend** is **NodeJS** (hosted on CodeSandbox).
-- **Frontend** is plain **HTML & Javascript**.
-
-### Usage Explanation
+## Requirements
```
Comment (rizdaprasetya) [inline en/snap/interactive-demo.md:20]: > Requirements

Is not quite the correct term. This section is not explaining what is required, it is simply explaining what "technology stack" being used for the demo.

It is a demo, nothing is required. It is explaining what the demo use. It is like when you see an Painting Demonstration, and the Painter tell you what he used to do the Painting, he explains that he use "water-color paint, canvas, brush, etc.". He does not require you.

File: en/snap/interactive-demo.md [hunk 2]
```diff
@@ -58,34 +56,34 @@ script.setAttribute('data-element-id','snap-backend-demo');
...
 --- | ---
-Card Number | 4811 1111 1111 1114 
-CVV | 123 
-Exp Month | Any month (e.g: `02`)
-Exp Year | Any future year (e.g: `2025`)
-OTP/3DS | 112233 
+Card Number | `4811 1111 1111 1114`
+CVV | `123`
+Exp Month | Any month in MM format. For example, `02`.
+Exp Year | Any future year, in YYYY format. For example, `2025`.
```
[the remark this answers is not in the document]
Comment (rizdaprasetya) [reply en/snap/interactive-demo.md:81]: yes this is good 👍

File: en/snap/interactive-demo.md (hunk 1 above)
Comment (rizdaprasetya) [reply en/snap/interactive-demo.md:20]: Nice, other than the feedback above, everything else looks good for this page 👍

File: en/snap/overview.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
 # Snap Overview
```
Comment (rizdaprasetya) [inline en/snap/overview.md:1]: Nice, this page looks good 👍

File: en/snap/preparation.md [hunk 4]
```diff
@@ -1,14 +1,14 @@
 # Preparation Before Integrating with Snap
```
Comment (rizdaprasetya) [inline en/snap/preparation.md:1]: This page is good

**Comment by rizdaprasetya** [comment]
Another note: looks like you bringing in other changes from your Core API submission, which I have to manually exclude. next time, if you are creating another PR for another group of files, please create the branch from your master branch. Not your other submission branch.

Anyway since the revision is small, and it is easier for me to fix it myself, I have [fixed and merged this PR submission here](https://github.com/Midtrans/technical-documentation-site/pull/28/commits/5779e1a33e50a91f315689410e43116f48ec0fcd)
Will close this now.
Thanks.

**Pull request closed without merging**